README.md

```diff
@@ -104,6 +104,14 @@ defmodule MyApp.Factory do
     }
   end
 
+  # dependent factory
```

Comment:
I think `# Modify existing factories` might be more clear here. What do you think?